sns_message_validator/sns_message_validator.py

```diff
@@ -24,6 +24,7 @@ def __init__(self,
                  signature_version='1'):
         self._cert_url_regex = cert_url_regex
         self._signature_version = signature_version
+        self.certSession = CachedSession('cert_cache', backend='memory')
```

Comment:
Nitpick: could you please rename the variables according to the pip8 naming standard?
`self._cert_session` would do. 
Thank you.

Reply:
Perfect! It's fixed in the next commit.